docs/Lifecycle/SDLRelease.md

```diff
@@ -0,0 +1,24 @@
+# **SDL Release**
+Successful iterative development require frequent release of stable product versions. Therefore SDL release process is independent from [SDL Contribution][SDL-CONT-LINK] process, however can be planned as subsequent step of it, if needed.
+
+## **Scope**
+In SDL a new release can be requested at any time and might be result of important feature implementation or set of sever hotfixes. SDL release include GitHub branch creation, development (branch stabilization), continuous integration (CI) and continuous deployment (CD) activities.
+
+## **Release process**
+At the moment of the release request not all features might be implemented. Therefor [CCB][CCB-LINK] first has to organize feature implementation according [SDL Contribution][SDL-CONT-LINK] process and only after prepare release branch.<br>
```

Comment:
Maybe we need to specify a Release version during CBB approval? (and mention it in CBB as a part of Owners/Group manger responsibility?)
IT will helps to avoid situation when someone merge feature of Next major feature to develop and mess up releasing next minor one.